packages/storefront-lwc/scripts/runtime.js

```diff
@@ -105,6 +105,7 @@ function validateConfig(config) {
         sess.cookie.secure = true; // serve secure cookies
     }
 
+    sampleApp.expressApplication.disable('x-powered-by');
```

Comment:
This is unrelated, but it is generally security best practice to not tell the client what is powering the API.